core/autoscaling.go

```diff
@@ -8,6 +8,13 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/autoscaling"
 	"github.com/aws/aws-sdk-go/service/ec2"
+	"math"
+)
+
+const (
+	OnDemandPercentageLong  = "on_demand_percentage"
+	OnDemandNumberLong      = "on_demand_number"
```

Comment:
Sure, can be changed easily. Regarding the single tag combined, we could think about it, when adding more options, but for the 'on-demand' configuration where only one value has to be taken from whether percentage, whether number, it did not make that much sense to implement it.

Considering the number of tags allowed in ASG, I don't know if we will implement it, otherwise something like this could be a good base: https://play.golang.org/p/6Q8iV3paZx